.github/workflows/build.yml

```diff
@@ -0,0 +1,135 @@
+name: continuous integration
+
+on: [push, pull_request]
```

Comment:
I think we want this to be like https://github.com/hyperium/tonic/blob/master/.github/workflows/CI.yml#L3

Reply:
That's pretty much the same as the current, but you lock out other branches than master.

I think the whole point for a CI server to protect the master branch. Which you can only do, if you check those branches which will be merged to master. So, I would recommend to keep it like this. What do you think?